tests/test_events.py

```diff
@@ -23,14 +33,23 @@ def test_post_two_different_kids(
     auth_keys = [{"kid": "kid1", "secret": "secret1"},
                  {"kid": "kid2", "secret": "secret2"}]
     client = client_factory(auth_keys)
+    body = [{
```

Comment:
Same comment as above that you can probably just do `body=[]` for this test case.